### PR TITLE
move -cli crates to examples

### LINKED ISSUE #91: Remove the two `-cli` packages
Both of these should be converted into `examples`. The `push-cli` package may actually already basically be there.

### PR DESCRIPTION
**Pull request by JustusFluegel**
Closes #91

### CODE REVIEW

**Comment by JustusFluegel** [comment]
This in conjunction with #83 should also Close #65

**Review by NicMcPhee — APPROVED** [review]

**Comment by NicMcPhee** [comment]
My sense is that it will be easier to wait on this until we've merged in #83 (and #101 before that) before merging this in. These are great changes, though, and we definitely want to get them merged in soon!

**Pull request merged**
